packages/vega-schema/src/axis.js

```diff
@@ -151,6 +153,7 @@ const axis = object({
 
 export default {
   refs: {
+    axisOrient,
```

Comment:
Why is this added as a ref? We use it directly in the axis definition, and I don't see a reference to this schema created and used elsewhere.

Reply:
Forgot about this—removed!